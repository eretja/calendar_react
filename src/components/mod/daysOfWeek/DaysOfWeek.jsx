import React from "react";
import PropTypes from "prop-types";
import styles from "./daysOfWeek.scss";

/**
 * General component description in JSDoc format. Markdown is *supported*.
 */
const DaysOfWeek = (props) => (
  <div className={styles.calendar__weekDay}>
    <div>M</div>
    <div>T</div>
    <div>W</div>
    <div>T</div>
    <div>F</div>
    <div>S</div>
    <div>S</div>
  </div>
);

DaysOfWeek.propTypes = {
  /** Description of prop "text". */

  optionsState: PropTypes.string
};

DaysOfWeek.defaultProps = {
  text: "defaultText"
};

export default DaysOfWeek;
