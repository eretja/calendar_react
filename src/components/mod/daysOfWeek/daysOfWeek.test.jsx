import Adapter from "enzyme-adapter-react-16";
import React from "react";
//import renderer from "react-test-renderer";
import { shallow, mount, configure } from "enzyme";
import toJson from "enzyme-to-json";
import DaysOfWeek from "./DaysOfWeek";
//uimport "jest-styled-components";

configure({ adapter: new Adapter() });

describe("<DaysOfWeek /> enzyme snapshot", () => {
  it("serializes the styles", () => {
    const output = shallow(<DaysOfWeek name="Test" />);
    expect(toJson(output)).toMatchSnapshot();
  });
});