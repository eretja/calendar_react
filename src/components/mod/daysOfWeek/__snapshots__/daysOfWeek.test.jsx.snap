// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DaysOfWeek /> enzyme snapshot serializes the styles 1`] = `
<div
  className="calendar__weekDay"
>
  <div>
    M
  </div>
  <div>
    T
  </div>
  <div>
    W
  </div>
  <div>
    T
  </div>
  <div>
    F
  </div>
  <div>
    S
  </div>
  <div>
    S
  </div>
</div>
`;
